velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -243,10 +243,10 @@ void PageReader::prepareDataPageV1(const PageHeader& pageHeader, int64_t row) {
   auto pageEnd = pageData_ + pageHeader.uncompressed_page_size;
   if (maxRepeat_ > 0) {
     uint32_t repeatLength = readField<int32_t>(pageData_);
-    repeatDecoder_ = std::make_unique<arrow::util::RleDecoder>(
+    repeatDecoder_ = std::make_unique<::arrow::util::RleDecoder>(
```

Comment:
@yingsu00 why do we need this namespace change? Can you share the error message and the environment?

Reply:
I don't have the error messages anymore. It happened a couple of times on my Macbook(Intel chip). I also think it's easier this way to identify the namespace to be an external lib.